app/controllers/users_controller.rb

```diff
@@ -4,6 +4,11 @@ class UsersController < ApplicationController
 
   load_and_authorize_resource except: [:index, :show]
 
+  # limit is set on conference attendances
```

Comment:
It looks kinda weird to me to rescue these model-validation-related exceptions in a controller @F3PiX. I guess moving this closer to the User model and its validations would make more sense? 

You could either pack it into some `before_save` or whatever callbacks... Or you just don't use the `limit` option for the association but instead just a "normal validation.
```ruby
validates :attendances, length: { maximum: 3 }
```
should work as well I think.

However, since submitting an invalid number of conference attendances is already prevented on client side and then fully enforced on DB level (because save will never happen if the error is raised), I guess its also totally fine to give the client a 500. Because in this case *"something really went wrong"*, e.g. someone by hand manipulated the form, the JS is not working, etc. And these are probably errors we should not cover but act upon.